src/components/Menu/Menu.component.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { MenuButton, MenuContainer } from './Menu.styles';
+
+import './Menu.styles.js';
```

Comment:
If you're importing the components, there's no need to import the file `Menu.styles.js`.
So it would be only
```
import { MenuButton, MenuContainer } from './Menu.styles';

const Menu = () => (
.....
```